sempyro/hri_dcat/hri_data_service.py

```diff
@@ -168,6 +168,15 @@ class HRIDataService(DCATDataService):
         }
     )
 
+    modification_date: str = Field(
+        default=None,
+        description="Date on which the resource was changed.",
+        json_schema_extra={
+            "rdf_term": DCTERMS.modified,
+            "rdf_type": "xsd:dateTime"
+        }
```

Comment:
I've left the `modified` property out because it was defined in `DCATResource` already, but I see the `description` and `rdf_type` differ. What's the convention: always override?